packages/prediction-app/src/components/BacktestsTable/BacktestsTable.tsx

```diff
@@ -0,0 +1,235 @@
+import React from 'react';
+import {
+    DataTable,
+    DataTableHead,
+    DataTableRow,
+    DataTableBody,
+    DataTableCell,
+    DataTableColumnHeader,
+    Checkbox,
+    Button,
+    IconAdd16,
+    DataTableFoot,
+    Pagination,
+    Chip,
+} from '@dhis2/ui';
+import i18n from '@dhis2/d2-i18n';
+import {
+    createColumnHelper,
+    flexRender,
+    getCoreRowModel,
+    useReactTable,
+    getSortedRowModel,
+    getFilteredRowModel,
+    getPaginationRowModel,
+} from '@tanstack/react-table';
+import { BackTestRead, ModelSpecRead } from '@dhis2-chap/chap-lib';
+import { useNavigate } from 'react-router-dom';
+import styles from './BacktestsTable.module.css';
+import { BacktestActionsMenu } from './BacktestActionsMenu';
+import { BacktestsTableFilters } from './BacktestsTableFilters';
+import { BatchActions } from './BatchActions';
+
+const statuses = {
+    COMPLETED: 'completed',
+    RUNNING: 'running',
+    FAILED: 'failed',
+}
+
+const labelByStatus = {
+    [statuses.COMPLETED]: i18n.t('Completed'),
+    [statuses.RUNNING]: i18n.t('Running'),
+    [statuses.FAILED]: i18n.t('Failed'),
+}
+
+const columnHelper = createColumnHelper<BackTestRead & { status?: string }>();
+
+const columns = [
+    columnHelper.accessor('id', {
+        header: i18n.t('ID'),
+        filterFn: 'equals',
+    }),
+    columnHelper.accessor('name', {
+        header: i18n.t('Name'),
+        filterFn: 'includesString',
+    }),
+    columnHelper.accessor('created', {
+        header: i18n.t('Created'),
+        cell: (info) => info.getValue() ? new Date(info.getValue()!).toLocaleString() : undefined,
+    }),
+    columnHelper.accessor('modelId', {
+        header: i18n.t('Model'),
+        filterFn: 'equals',
+    }),
+    columnHelper.accessor('status', {
+        header: i18n.t('Status'),
+        filterFn: 'equals',
+        enableSorting: false,
+        cell: (info) => info.getValue() ? (
+            <Chip dense>
+                {labelByStatus[info.getValue() as keyof typeof labelByStatus]}
+            </Chip>
+        ) : null,
```

Comment:
Minor, but could work around type assertion by having a variable for the value
```suggestion
     cell: (info) => {
            const value = info.getValue()
            return value ? <Chip dense>{labelByStatus[value]}</Chip> : null
        }
```